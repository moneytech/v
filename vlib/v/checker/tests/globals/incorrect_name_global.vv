__global A := 1
