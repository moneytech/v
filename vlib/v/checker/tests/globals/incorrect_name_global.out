vlib/v/checker/tests/globals/incorrect_name_global.v:1:1: error: global name `A` cannot contain uppercase letters, use snake_case instead
    1 | __global A := 1
      | ~~~~~~~~~~